doc/sphinx/source/developer_guide/abi.rst

```diff
@@ -49,58 +82,68 @@ Include header and declare an ``abi_event`` as a global ``static`` variable (but
+
 In the initialization function (or later) call the binding function for the message you want to receive.
 
 .. code-block:: C
 
     AbiBindMsgDATA(ABI_BROADCAST, &ev, data_cb);
 
-
 The first parameter is the sender ID you want to receive the message from.
+Senders IDs can be found in the file ``sw/airborne/modules/core/abi_sender_ids.h``
 
-* ``ABI_BROADCAST`` is used to receive messages from all senders.
-* ``ABI_DISABLE`` disable the callback (it will never be called).
-* Senders IDs can be found in the file ``sw/airborne/modules/core/abi_sender_ids.h``
-
-The second parameter is a pointer to the global ``abi_event`` you declared. This variable **can't** be reused for another bind. You must declare one abi_event per bind.
+The second parameter is a pointer to the global ``abi_event`` you declared. This variable **can't** be reused for another bind.
+You must declare one abi_event per bind.
 
 The last parameter is your callback.
 
 .. note:: When creating your module with the Module Creator (in the Paparazzi Center : Tools -> Module Creator), you can add subcriptions to ABI messages.
 
+
 Airborne code for publisher
 ---------------------------
 
 Include header, then call the send function with the appropriate parameters
 
 .. code-block:: C
 
+    #include "modules/core/abi.h"
+
     float var = 2.;
     struct bla s;
     struct foo f;
     AbiSendMsgDATA(SENDER_ID, var, s, &f);
 
 Replace ``SENDER_ID`` by your sender ID defined in ``sw/airborne/modules/core/abi_sender_ids.h``.
 
-Your sender ID identifier should be constructed as the concatenation of the name of the message and the name of your module, suffixed with ``_ID``.
 
-.. admonition:: example
+Creating a new message
+------------------------
 
-    A good sender ID for a module ``toto`` sending the message ``DATA`` may be ``DATA_TOTO_ID``
+Creating a new message requires multiple files to be edited. Here are all the steps:
 
-.. warning::
+#. In ``conf/abi.xml`` find an unused message id and define a new message block as described in `Message definition`_
 
-    The values 0 and 255 are reserved for ``ABI_DISABLE`` and ``ABI_BROADCAST`` and thus shall not be used.
-    
-    You must also avoid using an ID already used to send the same message type.
+#. In ``sw/airborne/modules/core/abi_sender_ids.h`` define a ``SENDER_ID`` for your message as described in `Message definition`_
+
+#. Run ``make clean`` and ``make`` in the ``paparazzi`` folder
+
+#. In your module .c file where the new message should be sent, include the header ``"modules/core/abi.h"``, and call
+the function ``AbiSendMsg<YOUR_MESSAGE_NAME>(<YOUR_SENDER_ID>, ...)`` with the appropriate function parameters corresponding
+to the message fields, as shown in `Airborne code for publisher`_.
+
+#. To receive this message in another module .c file, follow the steps described in `Airborne code for subscriber`_
 
```

Comment:
When creating a new module with the GUI tool (from the menu Tools -> Module Creator), you can specify the ABI bindings that you will need, and it will generate the callbacks with the correct prototypes. I'm not really sure we should mention this, since this tool is not really documented yet, but it is still a nice feature when starting a new module.